### PR TITLE
more output when unregistered collections are used within a transaction

### PR DESCRIPTION
**Pull request by hkernbach**

### CODE REVIEW

File: arangod/Transaction/Manager.cpp [hunk 1]
```diff
@@ -353,7 +353,13 @@ Result Manager::createManagedTrx(TRI_vocbase_t& vocbase,
       if (cid == 0) {
         return false;
       }
-      state->addCollection(cid, cname, mode, /*nestingLevel*/0, false);
+      Result res = state->addCollection(cid, cname, mode, /*nestingLevel*/ 0, false);
+      if (res.errorNumber() == TRI_ERROR_TRANSACTION_UNREGISTERED_COLLECTION) {
+        // special error message to indicate which collection was undeclared
+        LOG_TOPIC("da1a8", WARN, arangodb::Logger::TRANSACTIONS)
```
Comment (jsteemann) [inline arangod/Transaction/Manager.cpp]: Mhhh. I don't like logging a message here. First of all, nobody will look into the logs, and even if, it is completely unclear to what operation the warning has belonged to.
Can't we just return the error to the end user?

Comment (hkernbach) [reply arangod/Transaction/Manager.cpp]: I originally wanted to do so, but there are functions returning, returning and returning. I don't want to interrupt/change the flow there.

Comment (jsteemann) [reply arangod/Transaction/Manager.cpp]: Yes, but a log warning is still useless here. Shall I fix it?

**Comment by jsteemann** [comment]
I think this PR is obsolete now, as the problem was solved differently.

**Pull request closed without merging**